src/Minsk.Generators/SyntaxNodeGetChildrenGenerator.cs

```diff
@@ -90,38 +91,30 @@ public void Execute(SourceGeneratorContext context)
                 indentedTextWriter.WriteLine("}");
 
                 indentedTextWriter.Flush();
+                stringWriter.Flush();
```

Comment:
Why is this needed?

Reply:
Don't we need to flush before `.ToString()`?

Reply:
I don't think so, because they are chained but it also doesn't hurt.